content/projects/observation_visualisations_app.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Interactive desktop application for visualizing camera trap data
+subtitle: Conservation
+summary: An open-source desktop application for visualizing camera trap data to support conservation efforts.
+clients:
+  - name: OSI Panthera
+    link: https://www.osi-panthera.org/
+    logo: /images/clients/osi-panthera/logo.jpg
+  - name: OSI
+    link: https://www.science-camps.com/
+    logo: /images/clients/osi/logo.png
+tools:
+  - React
+  - Electron
+  - Mapbox
+  - Camtrap DP
+status: fundraising
+pinned: true
+date: 2025-02-20
+image: /images/projects/observation_visualisations_app/cover.jpg
```

Comment:
The image is not displayed in the branch build: https://deploy-preview-5--thunderous-daffodil-0e2415.netlify.app/projects/observation_visualisations_app/

![image](https://github.com/user-attachments/assets/0a817e3d-3b45-4a08-be42-3c74ad6b6ec3)


![image](https://github.com/user-attachments/assets/57c23e1a-718f-4274-9c3a-44b7fe5ef699)

Reply:
It's working locally. Any idea what could go wrong? URL seems correct. Only difference is most of the cover are png and mine is jpeg but I don't see why it would be an issue